docker/reverse_proxy/nginx-staging.conf

```diff
@@ -31,7 +31,7 @@ server {
     server_name api.staging.ehrenamtskarte.app;
 
     location / {
-        set $allowed_origin 'https://druckerei.staging.ehrenamtskarte.app';
+        set $allowed_origin '*';
```

Comment:
Why do we want to allow all origins? Can't we just allow the druckerei and the ttf download?

Reply:
So we would need to change this to 'https://druckerei.staging.ehrenamtskarte.app http://localhost:7000'`. Otherwise the staging server is not usable from a dev environment. Because this is not security critical I would just allow any host+port to tuse the API.

Reply:
I thought browsers would always allow localhost as an origin?

Reply:
I think security-wise, it may be useful to prevent other websites to use our API (to prevent users from using other third-party websites as a client, we also prevent phishing passwords through these third parties (?) )

Reply:
Yes absolutely but this is only the staging server. I would say usability goes before security here.

Reply:
> I thought browsers would always allow localhost as an origin? 

CORS is also active on localhosz

Reply:
> Yes absolutely but this is only the staging server. I would say usability goes before security here.

Ah, yeah, true!